File: csharp/MapleStory.OpenAPI/Src/Dto/UnionRaiderInnerStatDTO.cs
using Newtonsoft.Json;

namespace MapleStory.OpenAPI.Dto
{
    /// <summary>
    /// 유니온 공격대 배치 정보
    /// </summary>
    public class UnionRaiderInnerStatDTO
    {
        /// <summary>
        /// 공격대 배치 위치 (11시 방향부터 시계 방향 순서대로 0~7)
        /// </summary>
        [JsonProperty("stat_field_id")]
        public string StatFieldId { get; set; }

        /// <summary>
        /// 해당 지역 점령 효과
        /// </summary>
        [JsonProperty("stat_field_effect")]
        public string StatFieldEffect { get; set; }
    }
}

File: java/src/main/java/dev/spiralmoon/maplestory/api/msea/dto/union/UnionRaiderInnerStatDTO.java
package dev.spiralmoon.maplestory.api.msea.dto.union;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * Union raider deployment information
 */
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class UnionRaiderInnerStatDTO implements dev.spiralmoon.maplestory.api.common.dto.union.UnionRaiderInnerStatDTO {

    /**
     * Raider deployment position (0 to 7, clockwise from 11 o'clock)
     */
    @JsonProperty("stat_field_id")
    private String statFieldId;

    /**
     * Occupation effects over the area
     */
    @JsonProperty("stat_field_effect")
    private String statFieldEffect;
}
